common/src/main/java/tachyon/Constants.java

```diff
@@ -274,10 +274,16 @@
       "tachyon.worker.tieredstore.level%d.dirs.quota";
   /**
    * This format is used as a template to generate the property name for a given level. e.g.,
-   * {@code String.format(Constants.WORKER_TIERED_STORE_LEVEL_RESERVED_RATIO_FORMAT, level)}
+   * {@code String.format(Constants.WORKER_TIERED_STORE_LEVEL_HIGH_WATERMARK_RATIO_FORMAT, level)}
    */
-  public static final String WORKER_TIERED_STORE_LEVEL_RESERVED_RATIO_FORMAT =
-      "tachyon.worker.tieredstore.level%d.reserved.ratio";
+  public static final String  WORKER_TIERED_STORE_LEVEL_HIGH_WATERMARK_RATIO_FORMAT =
+      "tachyon.worker.tieredstore.level%d.watermark.high.ratio";
```

Comment:
Also, could you update `docs/Configuration-Settings.md` and `docs/Tiered-Storage-on-Tachyon.md` where the original property name was referenced.